xbmc/addons/PVRClient.h

```diff
@@ -723,9 +723,41 @@ namespace PVR
     bool IsPlayingEncryptedChannel(void) const;
     bool IsPlayingRecording(void) const;
     bool IsPlaying(void) const;
-    CPVRRecordingPtr GetPlayingRecording(void) const;
+
+    /*!
+     * @brief Set the channel that is currently playing.
+     * @param channel The channel that is currently playing.
+     */
+    void SetPlayingChannel(const CPVRChannelPtr channel);
```

Comment:
future PVR addons can play multiple channels at the same time. if an addon implementes inputstream, it knows the playing channels anyway. if it doesn't, I see not need for it to know.